modules/graphman-operation-explode.js

```diff
@@ -82,7 +82,7 @@ let type1Exploder = (function () {
 
         if (options.explodeKeys && key === "keys") {
             if (entity.p12) {
-                utils.writeFile(`${targetDir}/${filename}.p12`, atob(entity.p12));
+                utils.writeFileBinary(`${targetDir}/${filename}.p12`, Buffer.from(entity.p12, 'base64').toString('binary'));
```

Comment:
reconsider this like below:
`utils.writeFileBinary(`${targetDir}/${filename}.p12`, Buffer.from(entity.p12, 'base64'));`